From 986157a17dfa9d0cc71f2a4389c23771c5a4800e Mon Sep 17 00:00:00 2001
From: hienhuynh <hien.huynh.px@rvc.renenas.com>
Date: Mon, 9 Apr 2018 11:25:40 +0700
Subject: [PATCH 114/475] ARM: dts: r8a7742-iwg21d-q7: Add audio routes setting

This adds routes for Audio DMA transfer to/from SRC and DVC for
playback and capture.

	DMA	DMApp
[MEM] -> [SRC] -> [DVC] -> [SSI] -> [CODEC]

	DMA     DMApp
[CODEC] -> [SSI] -> [DVC] -> [SRC] -> [MEM]

Current sound driver is supporting:
SSI3 -> DVC1 -> SRC3
SSI4 <- DVC0 <- SRC4

Signed-off-by: hienhuynh <hien.huynh.px@rvc.renenas.com>
Signed-off-by: Hao Bui <hao.bui.yg@rvc.renenas.com>
---
 arch/arm/boot/dts/r8a7742-iwg21d-q7.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts b/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts
index 4b17f2e..7d509e0 100644
--- a/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts
+++ b/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts
@@ -165,8 +165,8 @@
 
 	rcar_sound,dai {
 		dai0 {
-			playback = <&ssi4>;
-			capture = <&ssi3>;
+			playback = <&ssi4 &src4 &dvc1>;
+			capture = <&ssi3 &src3 &dvc0>;
 		};
 	};
 };
-- 
2.7.4

